H5P.BranchedVideo/branched-video.css

```diff
@@ -307,29 +308,22 @@ button:focus {
   color: white;
   margin : 0;
   position: absolute;
-  background-color: #b7b7b7;
+  background-color: #111111;
   padding-left: 2px;
   padding-right: 2px;
   padding-top: 0px;
 }
 
 @font-face {
    font-family: tapestry-font;
-   src: url('fonts/palanquin/palanquin-semibold.ttf');
+   src: url('fonts/arial/arial.ttf');
 }
 
 .h5p-fullscreen{
   background-color: #FFFFFF;
 }
 
 .tapestry-video-preview-normal{
-  width: 100px;
-  position: absolute;
-  z-index: 100;
-  display: none;
-}
-
-.tapestry-video-preview-fullscreen{
   width: 200px;
   position: absolute;
   z-index: 100;
```

Comment:
Could you also add in `box-shadow: rgba(0, 0, 0, 0.5) 0 0 20px;` here please so it stands out a bit more?